src/Microsoft.Framework.PackageManager/Commands/InstallCommand.cs

```diff
@@ -23,31 +23,29 @@ public InstallCommand(AddCommand addCmd, RestoreCommand restoreCmd)
             _restoreCommand = restoreCmd;
         }
 
-        public Reports Reports { get; set; }
+        public ILogger Logger { get; set; }
```

Comment:
This could be tracked via a different bug, but I think it would be better if `dnu` used `Microsoft.Framework.Logging`. I think we could also start converting NuGet to use that interface as well. Or, we could just build an adaptor.